region/maridia/inner-pink/Aqueduct.json

```diff
@@ -1066,6 +1156,40 @@
         ]}
       ]
     },
+    {
+      "link": [4, 3],
+      "name": "Leave Shinecharged",
+      "notable": false,
+      "requires": [
+        "Gravity",
+        {"canShineCharge": {
+          "usedTiles": 33,
```

Comment:
I was a little confused by this one. Should it be 3->3 and a shorter runway? or fewer frames?

Reply:
This is the 4->4 strat, but you continue all the way to 3.
For the frame count, I liked just storing the shinespark on top of the sand and then sinking.
Quickdropping is a little faster but kinda awkward to do fast.
And there should be some leniency on these sand entries in case you need to do a spinjump entry.